File: Python3/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    s1, s2 = "",""
    
    for n in num_list:
        if n % 2: #짝수일 때
            s1 += str(n)
        else:
            s2 += str(n)
    
    return int(s1) + int(s2)

File: C++/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    string s1 = "", s2 = "";
    for(int n : num_list) {
        if(n % 2) s1 += to_string(n);
        else s2 += to_string(n);
    }
    
    return stoi(s1) + stoi(s2);
}